modules/addressing-cuda-oom-errors.adoc

```diff
@@ -0,0 +1,51 @@
+:_module-type: PROCEDURE
+
+[id="addressing-cuda-oom-errors-for-the-single-model-serving-platform_{context}"]
+= Addressing CUDA Out of Memory errors
+
+[role="_abstract"]
+
+In certain cases, depending on the model and hardware accelerator used, the TGIS memory auto tuning algorithm may underestimate the amount of GPU memory used to process long sequences. This can lead to CUDA OOM error responses from the model server. In such cases, we recommend updating or adding additional parameters in the TGIS model-serving runtime as described in the following procedure.
```

Comment:
A few things here:

- In this case, "auto tuning" is a compound adjective that describes "algorithm". Therefore, it should be  hyphenated to auto-tuning".
- Again, use "might" instead of "may".
- I think it should be "might underestimate the amount of GPU memory **needed**"?
- Don't use "this" without an object because it is ambiguous. In this case, I suggest "this miscalculation".
- The first use of an abbreviation in a module should spell out the word in full followed by the abbreviation in parentheses. Therefore, "Compute Unified Architecture (CUDA) out-of-memory (OOM) error responses".
- Never use "we". See https://www.ibm.com/docs/en/ibm-style?topic=word-usage#we. Instead, I suggest "To resolve these types of errors, you must update or add..."
- Add a comma before "as described in..."